test/reorderError.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+	<script src="../dist/qunit.js"></script>
+	<script>
+	// When failed test that checked test call count.
+	// Ref #907
+	// This issues has 2 case.
+	// First Case.
+	// 1. First test failed and second test successed in first module.
+	// 2. First test failed in second module.
+	// 3. Test count should be 3.
+	
+	// Given
+	var totalCount1 = 0;
+	sessionStorage["qunit-test-First fail module-should be fail"] = 1;
+	sessionStorage["qunit-test-Second fail module-should be fail"] = 1;
+	QUnit.done( function( details ) {
+		totalCount1 = details.total;
+		delete sessionStorage["qunit-test-First fail module-should be fail"];
+		delete sessionStorage["qunit-test-Second fail module-should be fail"];
+	});
+
+	// When
+	QUnit.module( "First fail module" );
+	QUnit.test( "should be success", function( assert ) {
+	    assert.equal( 1, 1 );
+	});
+	QUnit.test( "should be fail", function( assert ) {
+	    assert.equal( 1, 2 );
+	});
+
+	QUnit.module( "Second fail module" );
+	QUnit.test( "should be fail", function( assert ) {
+	    assert.equal( 1, 2 );
+	});
+	</script>
+	<script src="../dist/qunit.js"></script>
+	<script>
+	// Second Case.
+	// 1. First test failed in first module.
+	// 2. First test failed in second module.
+	// 3. Test count should be 2.
```

Comment:
I can definitely see value in it, but the important thing is to try for some coverage right now, which I think this achieves. A better method can come later.